dataflow/src/processing.rs

```diff
@@ -166,6 +166,7 @@ where
         replay_key_cols: Option<&[usize]>,
         domain: &prelude::DomainNodes,
         states: &prelude::StateMap,
+        shard: Option<usize>,
```

Comment:
Let's make this `usize`, not `Option<usize>`.

Reply:
👍